src/Microsoft.TemplateEngine.Abstractions/Engine/IEngineConfig.cs

```diff
@@ -13,5 +13,7 @@ public interface IEngineConfig
         IReadOnlyList<string> Whitespaces { get; }
 
         IDictionary<string, bool> Flags { get; }
+
+        string OperationIdFlag(int operationId);
```

Comment:
Yes, correct

From: Mike Lorbetske [mailto:notifications@github.com]
Sent: Tuesday, August 23, 2016 11:08 AM
To: seancpeters/microsoft-templateengine <microsoft-templateengine@noreply.github.com>
Cc: Sean Peters <sepeters@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [seancpeters/microsoft-templateengine] Refactor (#1)


In src/Microsoft.TemplateEngine.Abstractions/Engine/IEngineConfig.cs<https://github.com/seancpeters/microsoft-templateengine/pull/1#discussion_r75920491>:

> @@ -13,5 +13,7 @@ public interface IEngineConfig

>          IReadOnlyList<string> Whitespaces { get; }

>

>          IDictionary<string, bool> Flags { get; }

> +

> +        string OperationIdFlag(int operationId);

I think this one needs to be deleted?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/seancpeters/microsoft-templateengine/pull/1/files/a425b2f9e55c8aa446463bd90060cb6bc99db3d0#r75920491>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT8a8f6c5ZtOyARJAS2yztBqz6NhHZLOks5qizcagaJpZM4JqaeQ>.